template/src/ontology/Makefile.jinja2

```diff
@@ -101,9 +104,9 @@ PATTERNS = {% for x in project.pattern_group.products %} {{ x.id }}{% endfor %}
 PATTERNS = 
 {% endif %}
 PATTERN_ROOTS = $(patsubst %, $(PATTERNDIR)/%, $(PATTERNS))
-PATTERN_FILES = $(foreach n,$(PATTERN_ROOTS), $(n).owl)
+PATTERN_OWL_FILES = $(foreach n,$(PATTERN_ROOTS), $(n).owl)
 
-all_patterns: $(PATTERN_FILES)
```

Comment:
This goal, and the two preceeding lines, do not do anything currently (they are not called from anywhere). I think this should be deleted! (check the patterns: goal much later (somewhere around line 326), that is responsible for actually compiling the patterns.